modules/ROOT/pages/changelogs.adoc

```diff
@@ -3,15 +3,55 @@
 
 == Neo4j 2025.01
 **New:**
-[source, status codes, role="noheader"]
------
-Neo.ClientNotification.Cluster.ServerNotAvailable
-Neo.ClientNotification.Cluster.ServerCatchingUp
-Neo.ClientNotification.Cluster.ServerFailed
-Neo.ClientNotification.Cluster.ServerCaughtUp
------
-Starting from 2025.01, when using Cypher25, queries using `WAIT` return the notifications listed above instead of result rows.
-The behaviour for Cypher5 remains unchanged.
+[options="header", cols="<1m,<1"]
+|===
+| GQLSTATUS
+| Neo4j code
+
+| 01N80
+| Neo.ClientNotification.Cluster.ServerFailed
+
+| 01N81
+| Neo.ClientNotification.Cluster.ServerCatchingUp
+
+| 01N82
+| Neo.ClientNotification.Cluster.ServerNotAvailable
+
+| 03N85
+| Neo.ClientNotification.Cluster.ServerCaughtUp
+
+|===
+
+
+//Starting from 2025.01, when using Cypher25, queries using `WAIT` return the notifications listed above instead of result rows.
+//The behaviour for Cypher5 remains unchanged.
+
+== Neo4j 5.26
+
+**New:**
+
+[options="header", cols="<1m,<1"]
+|===
+
+| GQLSTATUS
+| Neo4j code
+
+| 01N72
+| Neo.ClientNotification.Statement.InsecureProtocol
+
+| 42I50
+| Neo.ClientError.Schema.TokenLengthError
+
+|===
+
+Starting from 5.26, Neo4j errors sent over drivers include new fields,
+such as GQLSTATUS code, StatusDescription, and DiagnosticRecord.
+For more information, see xref:errors/index.adoc#gqlstatus-error-object[GQL-status error object].
+
+
+**Deprecations:**
+
+The server-side Notification API and the `getNotifications()` method of the Result Core API were deprecated.
```

Comment:
Do we also want to mention earlier deprecations like NotificationCategory and the 5 deprecated Neo4j status codes?

Reply:
Now we have a specific page for deprecations and removals in the Operations manual. The preview is here https://development.neo4j.dev/docs/operations-manual/current/changes-deprecations-removals/.
Probably we need to include information about deprecations into this new page.
What do you think, @renetapopova?